command-line/command-line-interface.md

```diff
@@ -26,6 +26,34 @@ $ cd Downloads/
 - Go to www.npmjs.com/package/express, which is the Node Package Manager page for Express.js.
 - Everything in the block after the "$" is the command that you would type into the Command Line in order to install Express.js.
 
+### Set Up zsh
+First, if you purchased a Mac with the latest operating system, **Catalina**, *zsh* is already the default shell. Be sure to check by typing the following to your command line:
+
+```
+zsh --version
```

Comment:
Strictly speaking this just means you have zsh, not that it's the default. I think `echo $SHELL` might be better?

Reply:
Updated, thank you.